Sharpmake.Generators/VisualStudio/Csproj.Template.cs

```diff
@@ -39,6 +39,7 @@ public static class Project
 @"  <PropertyGroup>
     <Configuration Condition="" '$(Configuration)' == '' "">[options.DefaultConfiguration]</Configuration>
     <Platform Condition="" '$(Platform)' == '' "">[defaultPlatform]</Platform>
+    <Platforms>[defaultPlatform]</Platforms>
```

Comment:
Should it have a condition like the others? In case it's already set to sthg?

Reply:
Hmm, it's been a while since we added this one.
somehow I recall that you had to list available platforms there, and if that is empty then things didn't work

so I think the real solution is to have all platforms there


Would have to dig a bit to see how to do that though